### PR TITLE
Add coin icons on Florist store

### PR DESCRIPTION
**Pull request by yukari-n-erb**
## Description

Add coin icons on florist store pages.

## Closes issue(s)

https://github.com/ice-cream-the-fat-cat/my-niwa-frontend/issues/214

## Screenshots

Screen Shot 2021-09-20 at 15 43 24
Screen Shot 2021-09-20 at 15 43 31

## Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

### CODE REVIEW

**Review by kome12 — APPROVED** [review]
LGTM!

File: src/pages/Florist/component/dialog.tsx [hunk 1]
```diff
@@ -86,9 +88,17 @@ export const AlertDialog: React.FC<AlertDialogProps> = ({
...
         />
-        <DialogContentText className={classes.priceText}>
-          price : {selectFlower.price}
-        </DialogContentText>
+        <Grid container justifyContent="center">
+          <DialogContentText className={classes.priceText}>
+            price :
+          </DialogContentText>
+          {"    "}
+          <img src={CoinAsset} alt="" className={styles.coin} />
```
Comment (conalli) [inline src/pages/Florist/component/dialog.tsx:96]: I think maybe the coin should be on the right side of the price

File: src/pages/Florist/index.tsx [hunk 2]
```diff
@@ -275,7 +276,10 @@ export const Florist = () => {
                           !userData.balance || flower.price > userData.balance
                         }
                       >
-                        Buy: {flower.price}
+                        Buy:{" "}
+                        <img src={CoinAsset} alt="" className={styles.coin} />
```
Comment (conalli) [inline src/pages/Florist/index.tsx:280]: I think maybe the coin should be on the right side of the price

**Review by conalli — APPROVED** [review]

**Review by curtisahuang — APPROVED** [review]
Agree with Conall, but otherwise LGTM

We may want to discuss increasing font-size for the store overall.

**Review by daiHash — APPROVED** [review]
Thanks LGTM!
As Curtis commented small changes in the font size and also consistency in the text case would be good take a look

File: src/pages/Florist/component/dialog.tsx [hunk 3]
```diff
@@ -86,9 +88,17 @@ export const AlertDialog: React.FC<AlertDialogProps> = ({
...
           className={styles.notBoughtPicDialog}
         />
-        <DialogContentText className={classes.priceText}>
-          price : {selectFlower.price}
-        </DialogContentText>
+        <Grid container justifyContent="center">
+          <DialogContentText className={classes.priceText}>
+            price :
+          </DialogContentText>
+          {"    "}
```
Comment (daiHash) [inline src/pages/Florist/component/dialog.tsx:95]: Please add margin to element than adding space like this

**Pull request merged**